### PR TITLE
ZEN-28206: Wrong time format on Events/Triggers

### PR DESCRIPTION
**Pull request by lundybernard**
Wrong time format on Events/Triggers

### CODE REVIEW

File: Products/ZenModel/ZentinelPortal.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def setupRoles(self, p):
 
     def setupPermissions(self, p):
         # Set up some suggested role to permission mappings.
-        print "Starting setup of default permissions."
+        #print("Starting setup of default permissions.")
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/ZentinelPortal.py]: debug code?

File: Products/Zuul/facades/triggersfacade.py [hunk 2]
```diff
@@ -719,15 +761,22 @@ def importTriggers(self, triggers):
...
-        existingUsers = {"{} (User)".format(x.id): IGlobalIdentifier(x).getGUID() for x in self._dmd.ZenUsers.getAllUserSettings()}
-        existingUsers.update({"{} (Group)".format(x.id): IGlobalIdentifier(x).getGUID() for x in self._dmd.ZenUsers.getAllGroupSettings()})
+        existingUsers = {
+            "{} (User)".format(x.id): IGlobalIdentifier(x).getGUID()
+            for x in self._dmd.ZenUsers.getAllUserSettings()
+        }
+        existingUsers.update({
+            "{} (Group)".format(x.id): IGlobalIdentifier(x).getGUID()
+            for x in self._dmd.ZenUsers.getAllGroupSettings()}
+        )
```
Comment (jpeacock-zenoss) [inline Products/Zuul/facades/triggersfacade.py]: Have the closing curly brace inline with the closing parenthesis to improve the symmetry of the code, e.g.
```
existingUsers.update({
    ...
})
```

File: Products/Zuul/routers/triggers.py [hunk 3]
```diff
@@ -120,18 +129,24 @@ def getNotification(self, uid):
...
         data = self._getFacade().getRecipientOptions()
-        return DirectResponse.succeed(data=data);
+        return DirectResponse.succeed(data=data)
 
     # subscription windows
     @serviceConnectionError
     def getWindows(self, uid, **kwargs):
+        log.info('TRACER: triggerRouter.getWindows(%s)', uid)
         response = self._getFacade().getWindows(uid)
+        log.info('TRACER: triggerRouter.getWindows: response = %s', response)
```
Comment (jpeacock-zenoss) [inline Products/Zuul/routers/triggers.py]: These look like debug logging statements that shouldn't be in production code.

File: Products/ZenModel/ZentinelPortal.py (hunk 1 above)
Comment (lundybernard) [reply Products/ZenModel/ZentinelPortal.py]: removed commented code, and did a PEP8 cleanup

File: Products/ZenModel/ZentinelPortal.py [hunk 4]
```diff
@@ -13,49 +13,46 @@
...
 
 from Products.CMFCore.PortalObject import PortalObjectBase
 from Products.CMFCore.utils import getToolByName
 
 from Products.ZenUtils import Security, Time
-from Products.ZenUtils.Utils import prepId
 from Products.ZenUtils.deprecated import deprecated
-from Products.ZenUI3.browser.interfaces import IErrorMessage
 
+import ZenossSecurity as zs
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/ZentinelPortal.py]: The convention in other places has been to import the names.

On a side note, I generally discourage the aliasing of module names unless it's necessary to resolve a name conflict issue.

File: Products/ZenModel/ZentinelPortal.py [hunk 5]
```diff
@@ -228,85 +228,97 @@ def setupTools(self, p):
...
+        mp(zs.UPDATE_TRIGGER,
+            [zs.ZEN_MANAGER_ROLE, zs.OWNER_ROLE, zs.MANAGER_ROLE], 1)
+        mp(zs.VIEW_TRIGGER,
+            [zs.ZEN_MANAGER_ROLE, zs.OWNER_ROLE,
+             zs.MANAGER_ROLE, zs.ZEN_USER_ROLE],
+            1)
+        mp(zs.UPDATE_NOTIFICATION,
+            [zs.ZEN_MANAGER_ROLE, zs.OWNER_ROLE, zs.MANAGER_ROLE], 1)
+        mp(zs.MANAGE_NOTIFICATION_SUBSCRIPTIONS,
+            [zs.ZEN_MANAGER_ROLE, zs.OWNER_ROLE, zs.MANAGER_ROLE], 1)
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/ZentinelPortal.py]: Now I know why you went with the module alias approach.

**Pull request merged**